src/AndcultureCode.CSharp.Sitefinity.Core/AndcultureCode.CSharp.Sitefinity.Core.csproj

```diff
@@ -273,6 +273,18 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Xml" />
+    <Reference Include="Telerik.OpenAccess, Version=2020.0.428.1, Culture=neutral, PublicKeyToken=7ce17eeaf1d59342, processorArchitecture=MSIL">
```

Comment:
As mentioned further down in the review, I'm not sure we want to add these references to the Core repo, as it's going to spread an infrastructure concern across entire solutions, since this `Core` package will be used everywhere.

Unfortunately, they don't expose an interface outside of `Telerik.OpenAccess` that we can use here...